src/components/DevelopmentGuide/DisablingGlobalStyles.mdx

```diff
@@ -1,10 +1,15 @@
 import { Meta, Source } from "@storybook/blocks";
 
+import ComponentDocsHeader from "../../utils/components/ComponentDocsHeader";
 import Link from "../Link/Link";
 
 <Meta title="Development Guide/Disabling Global Styles" />
 
-# Disabling Global Styles
+<ComponentDocsHeader
+  category="Development Guide"
+  componentName="Disabling Global Styles"
+  summary="Recommendations for using Reservoir global styles"
```

Comment:
Just an observation, the name and summary feel like they conflict. What do you think?

Reply:
The name is intentional since Chakra adds their own reset CSS. The only real case we don't want this is in Vega so we are disabling those reset styles. It's not using our global styles. A better summary might be, "Removing Chakra-reset base CSS styles".